Content.Server/Traits/Assorted/ParacusiaComponent.cs

```diff
@@ -0,0 +1,27 @@
+namespace Content.Server.Traits.Assorted;
+
+[RegisterComponent]
+public sealed class ParacusiaComponent : Component
+{
```

Comment:
This should be in shared and the client should handle it, server doesn't need to worry about hallucinations as it's wasted bandwidth and cycles.